src/groupeddataframe/splitapplycombine.jl

```diff
@@ -448,31 +486,38 @@ combine(f::Union{Base.Callable, Pair}, gd::GroupedDataFrame; keepkeys::Bool=true
     combine(gd, f, keepkeys=keepkeys)
 combine(gd::GroupedDataFrame, f::Base.Callable; keepkeys::Bool=true) =
     combine_helper(f, gd, keepkeys=keepkeys)
+combine(gd::GroupedDataFrame, f::typeof(nrow); keepkeys::Bool=true) =
+    combine(gd, [nrow => :nrow], keepkeys=keepkeys)
 
 function combine(gd::GroupedDataFrame, f::Pair; keepkeys::Bool=true)
     # move handling of aggregate to specialized combine
-    if isagg(first(f) => (last(f) isa Pair ? first(last(f)) : last(f)))
+    f_from, f_to = f
```

Comment:
Isn't this called `incols` or `incolnames` elsewhere in the code? Starting with `f` makes it sound like a function.

Reply:
no - this is something different. I changed it to `p` for clarity.